Sources/Kitura/CodableRouter.swift

```diff
@@ -704,22 +402,258 @@ extension Router {
         return false
     }
 
-    private func isContentTypeJson(_ request: RouterRequest) -> Bool {
+    internal func join(path base: String, with component: String) -> String {
+        let strippedBase = base.hasSuffix("/") ? String(base.dropLast()) : base
+        let strippedComponent = component.hasPrefix("/") ? String(component.dropFirst()) : component
+        return "\(strippedBase)/\(strippedComponent)"
+    }
+}
+
+//
+// Building blocks for Codable routing
+//
+public struct CodableHelpers {
+    /**
+     * Check if the given request has content type JSON
+     *
+     * - Parameter request: The RouterRequest to check
+     * - Returns: True if the content type of the request is application/json, false otherwise
+     */
+    public static func isContentTypeJSON(_ request: RouterRequest) -> Bool {
         guard let contentType = request.headers["Content-Type"] else {
             return false
         }
-        return (contentType.hasPrefix("application/json"))
+        return contentType.hasPrefix("application/json")
+    }
+
+    /**
+     * Get the HTTPStatusCode corresponding to the provided RequestError
+     *
+     * - Parameter from: The RequestError to map to a HTTPStatusCode
+     * - Returns: A HTTPStatusCode corresponding to the RequestError http code
+     *            if valid, or HTTPStatusCode.unknown otherwise
```

Comment:
Nitpick: Should the code elements here be back-ticked?

Reply:
Yes! I just missed them. :(